src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
If that is the reasoning then many other files, including the CA bundle (/usr/local/share/certs/ca-root-nss.crt) should be rewritten from a know good source to insure their contents are correct.  That argument is still nonsensical.  And rewriting them provides no benefit.  The code for checking instead is simple.

If at every turn the thought was it's just a few writes that hardly matter.  Well they add up and before you know it the system takes many minutes to boot due to a bunch of unnecessary disk activity.  All for no benefit.